src/reducers/layers.js

```diff
@@ -101,22 +102,24 @@ const defaultLayers = [
     },
 ];
 
-const layers = (state = defaultLayers, action) => {
+const layers = (state, action) => {
+    const prevState = state || defaultLayers();
```

Comment:
Would `state = state || defaultLayers()` / `if(!state){state = defaultLayers()}` work or is that an anti-pattern?

Reply:
I think to reassign a function parameter is an anti-pattern. The problem assigning defaultLayers() in the function definition is that it's evaluated before the translation configuration is ready.